pkg/client/auditor/auditor.go

```diff
@@ -197,10 +200,30 @@ func (a *defaultAuditor) audit() error {
 			withError = true
 			return noErr
 		}
-		for _, val := range dbs.Databases {
-			a.databases = append(a.databases, val.Databasename)
+		a.databases = nil
+		for _, db := range dbs.Databases {
+			dbMustBeAudited := len(a.auditDatabases) <= 0
+			for _, dbPrefix := range a.auditDatabases {
```

Comment:
I think a map could be used for auditDatabases

Reply:
Since the filtering can be a prefix, a map wouldn't suffice.